RelationalAI/Rest.cs

```diff
@@ -41,21 +41,16 @@ public class Rest
 
         private readonly ILogger _logger;
 
-        public Rest(Context context, ILogger logger, IAccessTokenHandler accessTokenHandler)
+        public Rest(Context context, ILogger logger)
```

Comment:
Can we relax the ctor API here to default to null so the arg is not required at all for logger, now that clients are much more likely to use the direct Rest ctor, it'd be nice to not force ugly calls of Rest(ctx, null)